### ISSUE TITLE
Resourse leak after halting docker by timeout

### ISSUE THREAD

**Issue by kotee4ko** [out-of-tree/out-of-tree#23]
I don't sure, but as I can see - docker processes hangs and load CPU.

pts1:
![image](https://user-images.githubusercontent.com/62266948/103991138-f90b1a80-519a-11eb-8177-9aa5ed913735.png)

pts2:
![image](https://user-images.githubusercontent.com/62266948/103991198-07f1cd00-519b-11eb-97e2-ed8e4d4dd8fa.png)

I can share more log data if needed, but I don't understand where to find it, yet :)

**Comment by jollheef** [comment]
Hello,

`out-of-tree log --help`.

What distribution are you use and which version of qemu/docker/etc.? Is it stable reproducible?

**Comment by kotee4ko** [comment]
`Linux IS-73100 3.10.0-1062.9.1.el7.x86_64 #1 SMP Fri Dec 6 15:49:49 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux`

games@IS-73100:~/.out-of-tree/examples/kernel-module$ **hostnamectl && echo "\x0a" && qemu-system-x86_64 -version && echo "\x0a" && docker version&& echo "\x0a" && go version**
```
   Static hostname: IS-73100
         Icon name: computer-server
           Chassis: server
        Machine ID: f8bbd469a54143d6a0ee9a1d69e25b8c
           Boot ID: 8c8a5df19c8a4b1b99a9eb60cfa52090
  Operating System: CentOS Linux 7 (Core)
       CPE OS Name: cpe:/o:centos:centos:7
            Kernel: Linux 3.10.0-1062.9.1.el7.x86_64
      Architecture: x86-64
\x0a
QEMU emulator version 5.2.50 (v5.2.0-901-ge79de63)
Copyright (c) 2003-2020 Fabrice Bellard and the QEMU Project developers
\x0a
Client: Docker Engine - Community
 Version:           20.10.2
 API version:       1.41
 Go version:        go1.13.15
 Git commit:        2291f61
 Built:             Mon Dec 28 16:17:48 2020
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.2
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.13.15
  Git commit:       8891c58
  Built:            Mon Dec 28 16:16:13 2020
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.3
  GitCommit:        269548fa27e0089a8b8278fc4fc781d7f65a939b
 runc:
  Version:          1.0.0-rc92
  GitCommit:        ff819c7e9184c13b7c2607fe6c30ae19403a7aff
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
\x0a
go version go1.15.5 linux/amd64

```

**Comment by jollheef** [comment]
I can't reproduce the same issue on the fresh CentOS 7 installation.

By the way, I notice that out-of-tree doesn't work there with default SELinux rules.

**Comment by jollheef** [comment]
Fixed in d972bae547e51de9f362ef38cb38fc5490c5972d
